optionals/compat_gm/CfgVehicles.hpp

```diff
@@ -97,19 +97,6 @@ class CfgVehicles {
         EGVAR(dragging,dragDirection) = 0;
     };
 
-    // STATIC
-
-    class gm_ge_army_shelteraceII_repair_base;
-    class gm_ge_army_shelteraceII_repair: gm_ge_army_shelteraceII_repair_base {
-        EGVAR(repair,canRepair) = 1;
```

Comment:
do these have `transportRepair > 0`?
because we never zeroed it out
if not then we might want to leave the ace config